src/engine/shared/console.cpp

```diff
@@ -546,7 +547,8 @@ void CConsole::ExecuteLineStroked(int Stroke, const char *pStr, int ClientId, bo
 					Print(OUTPUT_LEVEL_STANDARD, "console", aBuf);
 				}
 			}
-			else if(pCommand->GetAccessLevel() >= m_AccessLevel)
+			// the fallback is needed for the rust tests
+			else if(!m_pfnCanUseCommandCallback || CanUseCommand(Result.m_ClientId, pCommand))
```

Comment:
```suggestion
			else if(CanUseCommand(Result.m_ClientId, pCommand))
```
Isn't it also necessary for the client console?

Can you move the fallback into the function (`CConsole::CanUseCommand`)?